vendor/priors/gwt.js

```diff
@@ -416,30 +422,47 @@ function setOPFSKV(filename, value, directory) {
     });
     return future;
 }
+function waitFor(pendingReadFuture, future) {
```

Comment:
why do we need waitFor? Can't we just use the .thenApply fo the future or equivalent for promise?